project/plugins.sbt

```diff
@@ -1,13 +1,7 @@
 addSbtPlugin("com.eed3si9n"        % "sbt-unidoc"               % "0.4.3")
-addSbtPlugin("com.github.gseitz"   % "sbt-release"              % "1.0.13")
-addSbtPlugin("io.crashbox"         % "sbt-gpg"                  % "0.2.1")
-addSbtPlugin("org.scalastyle"      % "scalastyle-sbt-plugin"    % "1.0.0")
-addSbtPlugin("org.scoverage"       % "sbt-scoverage"            % "1.6.1")
-addSbtPlugin("org.scala-js"        % "sbt-scalajs"              % "0.6.33")
+addSbtPlugin("org.scala-js"        % "sbt-scalajs"              % "1.1.0")
 addSbtPlugin("com.github.tkawachi" % "sbt-doctest"              % "0.9.6")
-addSbtPlugin("org.xerial.sbt"      % "sbt-sonatype"             % "3.9.2")
 addSbtPlugin("com.47deg"           % "sbt-microsites"           % "1.2.1")
-addSbtPlugin("com.dwijnand"        % "sbt-travisci"             % "1.2.0")
-addSbtPlugin("org.lyranthe.sbt"    % "partial-unification"      % "1.1.2")
 addSbtPlugin("org.scalameta"       % "sbt-scalafmt"             % "2.4.0")
 addSbtPlugin("org.portable-scala"  % "sbt-scalajs-crossproject" % "1.0.0")
+addSbtPlugin("com.codecommit"      % "sbt-spiewak-sonatype"     % "0.14-d60d2f3")
```

Comment:
Interesting

Reply:
It has a lot of utility stuff that I don't feel like redefining everywhere. :-) The warnings set *does* need a little polishing on 2.13 though, since everything changed flags.